### PR TITLE
docs: updates to gcp quickstart

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a - just following up here: 
https://github.com/mesosphere/dcos-docs-site/pull/4515

I was just on the late show to this PR above, and so ... anyway changes are here.

## Description of changes being made

Mostly just to formatting - even though it's a WIP for now, to render it nicely on the dev docs page - it sort of needs to have this formatting.

Also replaced the `gcp` with `gcloud` commands - so one less thing to do when this is all complete.

One thing we may have to add is a section like this:
https://docs.d2iq.com/dkp/konvoy/1.8/install/install-gcp/#permission-settings

This is more or less the gcloud equivalent to setting up the aws IAM policies: 
https://docs.d2iq.com/dkp/konvoy/2.2/choose-infrastructure/aws/iam-policies/

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4522.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mmcelroy91 — APPROVED** [review]

**Comment by mmcelroy91** [comment]
Thanks lovey!!!

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md [hunk 1]
```diff
@@ -41,12 +42,12 @@ Before starting the Konvoy installation, verify that you have:
...
 1.  Create an GCP Service Principal (SP) by running the following command:
 
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
 
     <p class="message--note"><strong>NOTE: </strong>If an SP with the name exists, this command will rotate the password.</p>
 
     ```bash
-    az ad sp create-for-rbac --role contributor --name "$(whoami)-konvoy" --scopes=/subscriptions/$(az account show --query id -o tsv)
+    gcloud ad sp create-for-rbac --role contributor --name "$(whoami)-konvoy" --scopes=/subscriptions/$(az account show --query id -o tsv)
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md:50]: FWIW, This entire command is wrong,  'ad sp ...' is the way you create a Service Account on AZURE, not GCP...

I suggest just removing the contents of all the shell excerpts in this document until you have a confirmed procedure from engineering.

FWIW, GCP service account creation looks *something like* this....

    export GCP_PROJECT=<account/project name> 
    export GOOGLE_APPLICATION_CREDENTIALS="$HOME/.gcloud/credentials.json"

    gcloud iam service-accounts create "$USER"
    gcloud projects add-iam-policy-binding $GCP_PROJECT --member="serviceAccount:$USER@$GCP_PROJECT.iam.gserviceaccount.com" --role=roles/editor
    gcloud iam service-accounts keys create $GOOGLE_APPLICATION_CREDENTIALS --iam-account="$USER@$GCP_PROJECT.iam.gserviceaccount.com"
    
    Here's the official docs on now to use the 'gcloud' command line tool for this task:   https://cloud.google.com/iam/docs/creating-managing-service-accounts#iam-service-accounts-create-gcloud


```suggestion
 <! NEED INSTRUCTIONS ON HOW TO CREATE A SP ON GCP>
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md [hunk 2]
```diff
@@ -23,9 +23,10 @@ Before starting the Konvoy installation, verify that you have:
 ## Configure GCP prerequisites
 
 1.  Log in to GCP:
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+
     ```bash
-    gcp login
+    gcloud login
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md:29]: The command would appear to be 'gcloud auth login':   https://cloud.google.com/sdk/gcloud/reference/auth/login

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md [hunk 3]
```diff
@@ -59,7 +60,7 @@ Before starting the Konvoy installation, verify that you have:
     ```
 
 1.  Set the required environment variables:
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
 
     ```bash
     export GCP_SUBSCRIPTION_ID="<id>"         # b1234567-abcd-11a1-a0a0-1234a5678b90
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md:66]:
```suggestion
<!NEED FINAL OUTPUT HERE>
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md [hunk 4]
```diff
@@ -69,7 +70,8 @@ Before starting the Konvoy installation, verify that you have:
     ```
 
 1.  Base64 encode the same environment variables:
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+
     ```bash
     export GCP_SUBSCRIPTION_ID_B64="$(echo -n "${GCP_SUBSCRIPTION_ID}" | base64 | tr -d '\n')"
     export GCP_TENANT_ID_B64="$(echo -n "${GCP_TENANT_ID}" | base64 | tr -d '\n')"
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md:77]:
```suggestion
<!NEED FINAL OUTPUT HERE>
```

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**